### ISSUE TITLE
Preference exclude list

### ISSUE THREAD

**Issue by Marco105** [yaddran/brackets-occurrences-marker#7]
I work with your extension and it's great, thank you.
Just one idea: could you write your preference exclude  with an array for legibility?

BTW

**Comment by yaddran** [comment]
Hi, I am glad that it is useful. Can you please explain better your idea? Thank you.

**Comment by Marco105** [comment]
That was just an idea of driving the list of exclude language. 

If you look at the **jslint.options** in your _defaultPreferences_ file you could see what i mean.
It return an object with key/value of each option (one by line).
I thank that could be an exemple for deal with your own.
As 

```
"thevirtualeuoccur.preferences.exclude": {
    "_audio": false,
    "_bash": false,
    ...
}
```

i think that could improve legibility! doesn't it?

**Comment by yaddran** [comment]
Ah, I understand now. You think that the preferences should be saved as an object instead of the line by line for each of the excluded languages. It will be done. Thank you.

**Comment by yaddran** [comment]
Solved in version v1.0.4
